setup.py

```diff
@@ -88,7 +88,8 @@
               'py_modules': ['pyearth.earth', 'pyearth._version'],
               'ext_modules': ext_modules,
               'classifiers': ['Development Status :: 3 - Alpha'],
-              'requires': ['numpy', 'scipy', 'sklearn']}
+              'install_requires': ['numpy', 'scipy', 'scikit-learn',
+                                   'sphinx_gallery']}
```

Comment:
requiring numpy and scipy install with pip is dangerous. I've had terrible experience
with automatic update of my numpy setup with pip. I'd keep numpy / scipy as requires
and not automatically installed if possible.